kolibri/core/assets/src/logging.js

```diff
@@ -14,8 +14,8 @@ class Logger {
       const name = methodName.toLowerCase();
       const logFunction = this.logger[name];
       if (logFunction) {
-        this[name] = param => {
-          return this.logger[name](param);
+        this[name] = (...params) => {
```

Comment:
This being a flyby, does that mean it's somehow connected to the perseus stuff (or more apparent with it?), or unrelated?

Reply:
I pass our logging library to Perseus so that we log errors consistently from it - however, for some the errors I was most interested in here, it was passing multiple arguments, so I needed to make this update to debug more effectively.